gapic/schema/wrappers.py

```diff
@@ -737,6 +738,27 @@ def field_headers(self) -> Sequence[str]:
 
         return next((tuple(pattern.findall(verb)) for verb in potential_verbs if verb), ())
 
+    @property
+    def http_opt(self) -> Optional[Dict[str, str]]:
+        """Return the http option for this method."""
+        http = self.options.Extensions[annotations_pb2.http].ListFields()
+        if len(http) < 1:
+            return None
+
+        http_method = http[0]
+        answer: Dict[str, str] = {
+            'method': http_method[0].name,
```

Comment:
Based on our offline discussion, let's s/`'method'`/`'verb'`/, since "method" is overloaded (service method, object method, HTTP method)

Reply:
If this be madness, there be method to it! 💀